### ISSUE TITLE
Confused about the code for computing supervised simcse loss

### ISSUE THREAD

**Issue by KarenMars** [jjljkjljk/SimCSE-Chinese#1]
Hi,

I have some problms in understanding the code for computing the supervised simcse loss in `model.py`:
```python 
def simcse_sup_loss(y_pred, device, lamda=0.05):
    """
    有监督损失函数
    """
    similarities = F.cosine_similarity(y_pred.unsqueeze(0), y_pred.unsqueeze(1), dim=2)
    row = torch.arange(0, y_pred.shape[0], 3)
    col = torch.arange(0, y_pred.shape[0])
    col = col[col % 3 != 0]

    similarities = similarities[row, :]
    similarities = similarities[:, col]
    similarities = similarities / lamda

    y_true = torch.arange(0, len(col), 2, device=device)
    loss = F.cross_entropy(similarities, y_true)
    return loss
```
in ` y_true = torch.arange(0, len(col), 2, device=device)`, I can't understand why the ground truth for simlarity of each anchor and its entailments and contradiction can be regardes as this. Looking forward to your reply, thank you very much!
